js/bybit.js

```diff
@@ -665,6 +825,24 @@ module.exports = class bybit extends Exchange {
         return this.milliseconds () - this.options['timeDifference'];
     }
 
+    async isUnifiedMarginEnabled (params = {}) {
+        const enableUnifiedMargin = this.safeValue (this.options, 'enableUnifiedMargin');
+        if (enableUnifiedMargin === undefined) {
+            const response = await this.privateGetUserV3PrivateQueryApi (params);
```

Comment:
@sc0Vu I'm not sure this is working correctly my account is unified but this returns `false` because I receive this response:

```
result:
{id: '364993', note: 'ccxt', apiKey: 'pi0XyfFkvXN1nYCryc', readOnly: '0', secret: '', …}
retCode:
'0'
retExtInfo:
null
retMsg:
'OK'
time:
'1669716906592'
```